core/api.cfc

```diff
@@ -222,7 +222,17 @@
 			<cfif application._taffy.settings.allowCrossDomain eq true>
 				<cfheader name="Access-Control-Allow-Origin" value="*" />
 			<cfelse>
-				<cfheader name="Access-Control-Allow-Origin" value="#application._taffy.settings.allowCrossDomain#" />
+				<!---
+					The Access-Control-Allow-Origin header can only have 1 value so we check to see if the Origin header is
+					in the list of origins specified in the config setting and parrot back the Origin header if so.
+				--->
+				<cfset local.domains = listToArray( application._taffy.settings.allowCrossDomain )>
```

Comment:
Might I suggest using delimiter `,; ` (comma, semicolon, and space) so that if any of the three is used it will split as expected?